src/Header/Header.css

```diff
@@ -11,6 +11,7 @@ h1 {
   line-height: 17px;
   margin: 41px 0px 0px 0px;
   color: #2e930c;
+  pointer-events: none;
```

Comment:
Yeah, I did this because I was tired of seeing the default cursor over what feels like it should be a logo. At the time, I figured that after we implemented Router, this styling would be removed and replaced with the default "pointer" cursor for a link that redirects to the homepage.

Reply:
The `pointer-events` can change some things tied to the functionality, I wonder if you could accomplish that same goal by editing the `cursor` instead